server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -569,81 +559,4 @@ private boolean evaluatePredicate(Block block, PredicateDefinition predicate) {
     return new PredicateEvaluator(this.applicantData, predicateGenerator)
         .evaluate(predicate.rootNode());
   }
-
-  /**
-   * Returns the {@link Path}s and their corresponding scalar answers to a {@link
-   * ApplicantQuestion}. Answers do not include metadata.
-   */
-  // TODO(#4872): remove this method.
-  private ImmutableMap<Path, String> getScalarAnswers(ApplicantQuestion question) {
```

Comment:
Now that the CSV isn't using AnswerData, the scalarAnswers field is no longer used and we can remove this.

Similar logic was added to CsvColumnFactory